src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json

```diff
@@ -47,7 +47,7 @@
       "type": "build"
     },
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "9.0.1-beta1",
```

Comment:
@javiercn a separate commit or a separate PR? Thought removing the `imports` was why we were doing this version change or at least one of the major points.